libtiledbsoma/src/soma/soma_scene.cc

```diff
@@ -46,30 +46,9 @@ void SOMAScene::create(
     std::optional<TimestampRange> timestamp) {
     try {
         std::filesystem::path scene_uri(uri);
-
-        SOMAGroup::create(ctx, scene_uri.string(), "SOMAScene", timestamp);
-        SOMACollection::create((scene_uri / "img").string(), ctx, timestamp);
-        SOMACollection::create((scene_uri / "obsl").string(), ctx, timestamp);
-        SOMACollection::create((scene_uri / "varl").string(), ctx, timestamp);
-
-        auto name = std::string(std::filesystem::path(uri).filename());
-        auto group = SOMAGroup::open(
-            OpenMode::write, scene_uri.string(), ctx, name, timestamp);
-        group->set(
-            (scene_uri / "img").string(),
-            URIType::absolute,
-            "img",
-            "SOMACollection");
-        group->set(
-            (scene_uri / "obsl").string(),
-            URIType::absolute,
-            "obsl",
-            "SOMACollection");
-        group->set(
-            (scene_uri / "varl").string(),
-            URIType::absolute,
-            "varl",
-            "SOMAColelction");
+        auto group = SOMAGroup::create(
+            ctx, scene_uri.string(), "SOMAScene", timestamp);
+        // TODO: Set extra metadata.
```

Comment:
Do you have a tracking task for this? Or did you intend to do this on this PR?

Reply:
Good catch. I just added a story: https://app.shortcut.com/tiledb-inc/story/59915

I looked at implementing the changes in this PR, but it was more work than I expected and is lower priority.

Reply:
Please put
```
https://app.shortcut.com/tiledb-inc/story/59915
```

right there in the `TODO` comment

Otherwise LGTM! :)